src/table/body-cell/disabled-inline-editor.tsx

```diff
@@ -108,7 +108,7 @@ export function DisabledInlineEditor<ItemType>({
                 trackRef={iconRef}
                 arrow={position => <Arrow position={position} />}
                 renderWithPortal={true}
-                zIndex={2000}
+                zIndex={7000} // When used in portal we need z-index to be higher than modal's
```

Comment:
I've prepared an `z-index` article update to mention the 7000: `DTe2AsqcXfNa`. Will review that with the team.

Reply:
The document needs updates, but the change can go forward, approved